app/client/components/ColumnTransform.ts

```diff
@@ -11,6 +11,7 @@ import {TableData} from 'app/client/models/TableData';
 import {FieldBuilder} from 'app/client/widgets/FieldBuilder';
 import {UserAction} from 'app/common/DocActions';
 import {Disposable, Observable} from 'grainjs';
+import isObject from 'lodash/isObject';
```

Comment:
```suggestion
import isPlainObject from 'lodash/isPlainObject';
```

isObject is not enough, it treats functions and arrays as objects.